Logger/IEntity.cs

```diff
@@ -3,5 +3,9 @@ public interface IEntity
 {
 
         // Place members here.
+        Guid Id { get; init; }
+        string Name { get; 
+        //set; 
+    }
```

Comment:
```suggestion
        string Name { get; }
```
One line is more clear